packages/codecs/package.json

```diff
@@ -0,0 +1,93 @@
+{
+    "name": "@solana/codecs",
+    "version": "2.0.0-development",
+    "description": "Helpers for creating transaction codecs",
+    "exports": {
+        "browser": {
+            "import": "./dist/index.browser.js",
+            "require": "./dist/index.browser.cjs"
+        },
+        "node": {
+            "import": "./dist/index.node.js",
+            "require": "./dist/index.node.cjs"
+        },
+        "react-native": "./dist/index.native.js",
+        "types": "./dist/types/index.d.ts"
+    },
+    "browser": {
+        "./dist/index.node.cjs": "./dist/index.browser.cjs",
+        "./dist/index.node.js": "./dist/index.browser.js"
+    },
+    "main": "./dist/index.node.cjs",
+    "module": "./dist/index.node.js",
+    "react-native": "./dist/index.native.js",
+    "types": "./dist/types/index.d.ts",
+    "type": "module",
+    "files": [
+        "./dist/"
+    ],
+    "sideEffects": false,
+    "keywords": [
+        "blockchain",
+        "solana",
+        "web3"
+    ],
+    "scripts": {
+        "compile:js": "tsup --config build-scripts/tsup.config.package.ts",
+        "compile:typedefs": "tsc -p ./tsconfig.declarations.json && node node_modules/build-scripts/add-js-extension-to-types.mjs",
+        "dev": "jest -c node_modules/test-config/jest-dev.config.ts --rootDir . --watch",
+        "prepublishOnly": "version-from-git --no-git-tag-version --template experimental.short",
+        "publish-packages": "pnpm publish --tag experimental --access public --no-git-checks",
+        "style:fix": "pnpm eslint --fix src/* && pnpm prettier -w src/* package.json",
+        "test:lint": "jest -c node_modules/test-config/jest-lint.config.ts --rootDir . --silent",
+        "test:prettier": "jest -c node_modules/test-config/jest-prettier.config.ts --rootDir . --silent",
+        "test:treeshakability:browser": "agadoo dist/index.browser.js",
+        "test:treeshakability:native": "agadoo dist/index.native.js",
+        "test:treeshakability:node": "agadoo dist/index.node.js",
+        "test:typecheck": "tsc --noEmit"
+    },
+    "author": "Solana Labs Maintainers <maintainers@solanalabs.com>",
+    "license": "MIT",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/solana-labs/solana-web3.js"
+    },
+    "bugs": {
+        "url": "http://github.com/solana-labs/solana-web3.js/issues"
+    },
+    "browserslist": [
+        "supports bigint and not dead",
+        "maintained node versions"
+    ],
+    "dependencies": {
+        "@solana/codecs-core": "workspace:*",
+        "@solana/codecs-data-structures": "workspace:*",
+        "@solana/codecs-numbers": "workspace:*",
+        "@solana/codecs-strings": "workspace:*"
+    },
```

Comment:
What about `@solana/options`?

Reply:
I was debating with myself whether or not to add it since it's a slightly higher level building block then the others but at the same time the Nullable codec is part of `codecs-data-structures` and does a very similar job.

The fact that you asked for it helped me understand that others also see it as part of the codecs family so I will add it to this umbrella package. ☺️